CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
+<!-- markdownlint-disable MD013 -->
```

Comment:
only one way to find out!

It gets overridden because the header gets recreated everytime, adding the comment line to the `.versionrc.json` file fixes this.
```json
    "header": "<!-- markdownlint-disable MD013 --># Changelog\n\nAll notable changes to this project will be documented in this file.",
 ```